README.md

```diff
@@ -0,0 +1,106 @@
+# Django Url-Shortener App
+
+## Setting up your Development Environment ##
+1. First, fork this repository to your account.
+
+2. Create a virtual environment on your machine. 
+    ```
+    virtualenv -p python3 env
```

Comment:
You can make it clear that the user can give environment name according to his or her wish. Better replace "env" with "environment name". Will look much better!